### PR TITLE
Change Dockerfile to be based on opendatacube/geobase

### PR DESCRIPTION
**Pull request by alexgleith**
- [x] makes the built image smaller
- [x] should mean that geo libraries are fully working together.

### CODE REVIEW

**Comment by jeremyh** [comment]
Looks good. That's a huge decrease in runtime -- it's much closer to what Travis was. Oddly, the running of the tests is much faster now, not just the machine creation.

**Review by jeremyh — APPROVED** [review]

File: requirements-docker.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+--extra-index-url https://packages.dea.ga.gov.au/
+digitalearthau
+datacube
```
Comment (Kirill888) [inline requirements-docker.txt:3]: @alexgleith for cloud installs `datacube[s3]`  is needed, but you probably want to enable `performance` and `distributed` as well. There are a lot of timestamps in the metadata and python date parsing is way too generic and hence slow, `performance` option installs a much faster date parsing lib and some other bits too. But it's not just about speed, majority of our environments use that lib so fallback code doesn't get tested all that much with real life work loads.

Comment (alexgleith) [reply requirements-docker.txt:3]: Distributed would be for Dask, right?

Explorer is not going to be using Dask, and doesn't actually do any data reading to s3.

So what about just installing with the `performance` flag? Or do you think we should install with all three more as a 'best practice' thing?

Comment (Kirill888) [reply requirements-docker.txt:3]: distributed is for distributed dask, dask itself is not optional.
s3 pulls in boto3, which is probably installed anyway, but yes if you do not read anything from s3 then you don't need it.
performance pulls in ciso8601 and bottleneck, so time parsing and "misc other", not sure how much impact bottleneck has or how it works for that matter.

**Pull request merged**